### PR TITLE
[ahoy.js] Update track methods to the latest version

### PR DESCRIPTION
**Pull request by sudhanshug16**
Update track methods to the latest version

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:
If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/ankane/ahoy.js/blob/3d9bb076ca3b938d2094cffeac44e23e427d567a/src/index.js#L446, 
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/ahoy.js/index.d.ts [hunk 1]
```diff
@@ -135,17 +132,14 @@ declare namespace ahoy {
     /** Send a single Ahoy tracking event. */
     function track(name: string, properties?: EventProperties): boolean;
 
-    /** Track all views and clicks. */
-    function trackAll(): void;
-
     /** Track form control changes. */
```
Comment (TimothyGu) [inline types/ahoy.js/index.d.ts:135]: Could you mark this function as deprecated? See https://github.com/ankane/ahoy.js/blob/a644e83ee99fde60c24cf2110b6b3f05a43f279a/src/index.js#L457

```suggestion
    /**
     * Track form control changes.
     * @deprecated trackChanges is deprecated and will be removed in 0.5.0
     */
```

**Review by TimothyGu — CHANGES_REQUESTED** [review]
Thanks! One minor comment.

File: types/ahoy.js/index.d.ts [hunk 2]
```diff
@@ -87,17 +87,14 @@ export function debug(enabled?: boolean): boolean;
 /** Send a single Ahoy tracking event. */
 export function track(name: string, properties?: EventProperties): boolean;
 
-/** Track all views and clicks. */
-export function trackAll(): void;
-
 /** Track form control changes. */
```
Comment (TimothyGu) [inline types/ahoy.js/index.d.ts:90]: Sorry, here too.

```suggestion
/**
 * Track form control changes.
 * @deprecated trackChanges is deprecated and will be removed in 0.5.0
 */
```

**Review by TimothyGu — CHANGES_REQUESTED** [review]

File: types/ahoy.js/index.d.ts (hunk 2 above)
Comment (sudhanshu16) [reply types/ahoy.js/index.d.ts:90]: np

**Comment by sudhanshu16** [comment]
Thanks for the quick review (wasn't expecting it)! Made the suggested changes.
You're awesome!

**Review by TimothyGu — APPROVED** [review]

**Comment by sudhanshu16** [comment]
Ready to merge

**Comment by sudhanshu16** [comment]
@TimothyGu do I have to do anything to trigger release of this package?
Might be dumb question but I couldn't find any documentation regarding the same.

**Comment by TimothyGu** [comment]
@sudhanshu16 Sorry I'm not sure, but I think @typescript-bot will automatically publish a new release. See "All packages are being published to npm in under an hour and a half": https://github.com/DefinitelyTyped/DefinitelyTyped#current-status

**Comment by TimothyGu** [comment]
In fact 0.4.0 is live for `@types/ahoy.js`: https://www.npmjs.com/package/@types/ahoy.js

**Pull request merged**